extensions/sql/asset/sql_asset_index_loader_proposal.md

```diff
@@ -0,0 +1,187 @@
+## ER Diagramm
+
+![ER Diagram](http://www.plantuml.com/plantuml/png/XP513e8m44NtFSMiJNe1GWWhDnwY6Ud4b40XdS58XBjRnI346dVD_DVVtpyb2mOPsaQH5oSZaAqCCfF0NG4Su7KspcR04fo_nA7MQbPVlB4eYDO6Olvn5_ByU2gAnU89zA7hAOWZpa3e9X6ekVCHIJtmfiU_xglnQ3osj8yE7t4PcoismmfjzUvSKoLxVHznBMkQj4vtiO7c22N-eqhoDgxW0G00)
+
+## Table schema
+
+```sql
+
+-- table: edc_asset
+CREATE TABLE IF NOT EXISTS edc_asset (
+    asset_id VARCHAR(255) NOT NULL,
+    deleted BOOLEAN DEFAULT FALSE,
```

Comment:
@bcronin90 This line also needs to be deleted if soft-delete is now within the scope of this PR.